### PR TITLE
Rename dask-kubernetes to dask-gke

### PR DESCRIPTION
**Pull request by mrocklin**
cc @martindurant

### CODE REVIEW

**Comment by martindurant** [comment]
Why the changes in dask_scheduler.yaml ?

**Comment by mrocklin** [comment]
Whoops!  I had had a bunch of other changes sitting in there that got merged in.  I've force pushed a cleaner commit.

File: setup.py [hunk 1]
```diff
@@ -7,10 +7,10 @@
 with open('README.md', encoding='utf-8') as f:
     readme = f.read()
 
-setup(name="dask-kubernetes",
+setup(name="dask-gke",
       version="0.0.1",
       description="Dask Kubernetes",
-      url="https://github.com/martindurant/dask-kubernetes",
+      url="https://github.com/martindurant/dask-gke",
```
Comment (martindurant) [inline setup.py:13]: This url should be /dask/

**Comment by mrocklin** [comment]
> Did you do any tests on this?

I did a basic create and delete operations.  Things seemed decent.   The cluster worked well on the examples.  I didn't go through each of the operations though.

> I assume that the change of the repository name (here and in forks) should follow immediately after merger.

Yes, I think so

**Comment by mrocklin** [comment]
@martindurant can I ask for another review?

**Comment by martindurant** [comment]
Looks good. I haven't had a chance to test anything yet, but I'm pretty confident it should all be OK.

**Pull request merged**